test('First test', () => {
    // throw new Error('Something wrong');
});